nixos/modules/security/doas.nix

```diff
@@ -266,14 +266,14 @@ in
             # completely replace the contents of this file, use
             # `environment.etc."doas.conf"`.
 
-            # "root" is allowed to do anything.
-            permit nopass keepenv root
-
             # extraRules
             ${concatStringsSep "\n" (lists.flatten (map mkRule cfg.extraRules))}
 
             # extraConfig
             ${cfg.extraConfig}
```

Comment:
We should update the docs of `extraRules` and `extraConfig` to explain that `permit nopass keepenv root` cannot be overridden (as it could be before by making it last-matched as you said). Just a simple "Be aware that the  rule allowing `root` to do anything will always be applied and never overridden." or something like that.

Reply:
Done.